dotCMS/src/main/java/com/dotmarketing/init/DotInitScheduler.java

```diff
@@ -755,7 +550,7 @@ private static void addSystemEventsJob () {
 	 */
 	private static void scheduleJob(final JobBuilder jobBuilder) {
 		try {
-			Scheduler sched = QuartzUtils.getStandardScheduler();
+			Scheduler sched = DotSchedulerFactory.getInstance().getScheduler();

```

Comment:
Above there are references to `QuartzUtils.getScheduler()`. I wonder if they are the same and i f they are, we should pick one.